cmd/frontend/graphqlbackend/external_services.go

```diff
@@ -158,28 +158,32 @@ func (r *schemaResolver) UpdateExternalService(ctx context.Context, args *update
 }
 
 type excludeRepoFromExternalServiceArgs struct {
-	ExternalService graphql.ID
-	Repo            graphql.ID
+	ExternalServices []graphql.ID
+	Repo             graphql.ID
 }
 
-// ExcludeRepoFromExternalService excludes given repo from given external service config.
-func (r *schemaResolver) ExcludeRepoFromExternalService(ctx context.Context, args *excludeRepoFromExternalServiceArgs) (*EmptyResponse, error) {
+// ExcludeRepoFromExternalServices excludes given repo from given external services configs.
```

Comment:
Same as above and other minor nits in one suggestion:

```suggestion
// ExcludeRepoFromExternalServices excludes the given repo from the given external service configs.
```